src/validator/FetchVerticesValidator.h

```diff
@@ -33,21 +33,17 @@ class FetchVerticesValidator final : public Validator {
     Status preparePropertiesWithoutYield();
     Status prepareProperties();
 
-    static const Expression* findInvalidYieldExpression(const Expression* root);
-
     // TODO(shylock) merge the code
     std::string buildConstantInput();
     std::string buildRuntimeInput();
 
 private:
-    GraphSpaceID spaceId_{0};
     DataSet srcVids_{{kVid}};  // src from constant
     Expression* srcRef_{nullptr};  // src from runtime
     Expression* src_{nullptr};  // src in total
-    std::string tagName_;
-    // none if not specified tag
-    folly::Optional<TagID> tagId_;
-    std::shared_ptr<const meta::SchemaProviderIf> schema_;
+    bool onStar_{false};
```

Comment:
suggest a meaningful name related to all tags

Reply:
There are two `allTags` in here, one extend from * , the other means user type all tags.